examples/3d/wireframe.rs

```diff
@@ -26,6 +26,10 @@ fn setup(
 ) {
     // To draw the wireframe on all entities, set this to 'true'
     wireframe_config.global = false;
+    // You can also change the default color of the wireframes, which is used only if `global` is set.
```

Comment:
I don't like coupling configuration like this: it leads to frustrations when users change a value and nothing happens.

Why can't we use this when `global` is false too? :)